tor/core/config.py

```diff
@@ -0,0 +1,128 @@
+# Load configuration regardless of if bugsnag is setup correctly
+try:
+    import bugsnag
+except ImportError:
+    bugsnag = None
+
+from tor import __version__
+
+
+class BaseConfig:
+    '''
+    '''
```

Comment:
So just to make sure I understand, these are deliberately blank because they get populated by `initialize()`, yah?